Classes/Service/SolrServiceProvider.php

```diff
@@ -1147,4 +1148,16 @@ protected function tagForFacet(string $facetID): string
     {
         return 'facet-'.$facetID;
     }
+
+    /*
+     * Set configured main query operator. Defaults to 'AND'.
+     */
+    private function addMainQueryOperator()
+    {
+        $mainQueryOperator = 'AND';
+        if (isset($this->settings['mainQueryOperator'])) {
+            $mainQueryOperator = $this->settings['mainQueryOperator'];
```

Comment:
How should a developer know about this setting? It's not documented anywhere.
So, please put this in the documentation as well, and maybe provide examples in the TypoScript code.
Do we have to evaluate/trim the setting or silently fail if the operator is not supported by Solr (e.g `FOO` as setting)?